app/src/main/java/org/wikipedia/notifications/NotificationActivity.kt

```diff
@@ -223,11 +223,13 @@ class NotificationActivity : BaseActivity() {
 
     private fun delimitedFilteredWikiList(): String {
         val excludedWikiCodes = Prefs.notificationExcludedWikiCodes
-        return NotificationsFilterActivity.allWikisList().filterNot { excludedWikiCodes.contains(it) }.joinToString("|") {
-            val defaultLangCode =
-                    WikipediaApp.getInstance().language().getDefaultLanguageCode(it) ?: it
-            "${defaultLangCode.replace("-", "_")}wiki"
-        }
+        val filteredWikiList =
+            (dbNameMap.keys.plus(NotificationsFilterActivity.allWikisList())).filterNot { excludedWikiCodes.contains(it) }.map {
```

Comment:
It feels not quite right here.

The `dbNameMap.keys` contains something like `commonswiki`, but the `NotificationsFilterActivity.allWikisList()` contains language codes without `wiki`. Looks like the `filterNot` will not filter correctly at this point.

Can we do something like: `(dbNameMap.keys.plus(NotificationsFilterActivity.allWikisList().map { (WikipediaApp.getInstance().language().getDefaultLanguageCode(it) ?: it) + "wiki" }`?

Please note that I didn't check the `zh-yue` if that also works for it.